auth-api/config.py

```diff
@@ -234,8 +234,8 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
     NOTIFY_API_URL = 'http://localhost:8080/notify-api/api/v1'
 
     # If any value is present in this flag, starts up a keycloak docker
-    USE_TEST_KEYCLOAK_DOCKER = os.getenv('USE_TEST_KEYCLOAK_DOCKER', None)
-    USE_DOCKER_MOCK = 'YES'
+    USE_TEST_KEYCLOAK_DOCKER = None
```

Comment:
These values get it from .env so that local can still run with pytest docker

Reply:
fixed